### PR TITLE
Allow the use of the legacy Dynamic Data implementation

### PR DESCRIPTION
**Pull request by javmorcas**

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -98,6 +98,8 @@ if NOT "%1"=="" (
 				SET RELEASE_DEBUG=debug
 		) ELSE if "%1"=="--dynamic" (
 				SET STATIC_DYNAMIC=dynamic
+		) ELSE if "%1"=="--legacy-DynamicData" (
+				SET LEGACY_DD_IMPL=1
```
Comment (softwarejimenez) [inline build.bat]: define before LEGACY_DD_IMPL:

```
set LEGACY_DD_IMPL=0
```

File: build.sh [hunk 2]
```diff
@@ -635,6 +641,9 @@ while [ "$1" != "" ]; do
         --secure)
             USE_SECURE_LIBS=1
             ;;
+        --legacy-DynamicData)
+            LEGACY_DD_IMPL=1
```
Comment (softwarejimenez) [inline build.sh]: define the variable first
```
LEGACY_DD_IMPL=0
```

File: build.bat (hunk 1 above)
Comment (javmorcas) [reply build.bat]: Thanks! Good point!

File: build.bat [hunk 3]
```diff
@@ -256,6 +258,11 @@ if !BUILD_CPP! == 1 (
 		set "additional_defines_custom_type= -D RTI_CUSTOM_TYPE=%custom_type%"
 	)
 
+	if !LEGACY_DD_IMPL! == 1 (
+        echo [INFO]: Allow the use of both legacy and new Dynamic Data Impl.
```
[the remark this answers is not in the document]
Comment (javmorcas) [reply build.bat]: I think the problem is that we are using tabs and here I added spaces. Fixed.

File: build.sh (hunk 2 above)
Comment (javmorcas) [reply build.sh]: Fixed

**Comment by javmorcas** [comment]
@softwarejimenez, I added your CR feedback

**Pull request merged**